packages/next/src/build/handle-externals.ts

```diff
@@ -234,7 +235,9 @@ export function makeExternalHandler({
       if (isExternal) {
         // it's important we return the path that starts with `next/dist/` here instead of the absolute path
         // otherwise NFT will get tripped up
-        return `commonjs ${localRes.replace(/.*?next[/\\]dist/, 'next/dist')}`
+        return `commonjs ${normalizePathSep(
+          localRes.replace(/.*?next[/\\]dist/, 'next/dist')
```

Comment:
I think so yeah, I don't know why that was added.

Reply:
The above comment says `it's important we return the path that starts with `next/dist/` here instead of the absolute path otherwise NFT will get tripped up`, since `normalizePathSep` can normalize the whole path into `/`, I think we can remove it.

Reply:
No that is different, normalizePathSep changes `\\` to `/` but it doesn't change absolute paths to module-relative paths.